packages/flutter_driver/lib/src/extension/extension.dart

```diff
@@ -29,6 +29,8 @@ import '../common/render_tree.dart';
 import '../common/request_data.dart';
 import '../common/semantics.dart';
 import '../common/text.dart';
+import '../common/wait.dart';
+import 'internal_wait.dart';
```

Comment:
internal_wait.dart is a strange name. Maybe just "wait_conditions.dart"?

Reply:
Had difficulty in coming up with a good name... Updated to wait_conditions.dart.